### ISSUE TITLE
Closed issues retain "status: waiting-for-triage" label

### ISSUE THREAD

**Issue by sbrannen** [spring-io/issue-bot#7]
If somebody closes an issue without removing the `status: waiting-for-triage` label, it stays in place.

This typically occurs if the issue was reviewed and summarily closed as "declined" or "invalid" (even without those labels being set).

Would it be possible to modify the bot so that the `status: waiting-for-triage` label is automatically removed from issues when they are closed?

**Comment by wilkinsona** [comment]
Yeah, that's possible. For ease of deployment, the Bot deliberately doesn't use web hooks, but it could poll for closed issues labelled as waiting for triage without much risk of running into any rate limits. It could then remove the label from any issues that it finds.

**Comment by sbrannen** [comment]
Sounds good. Thanks for the quick feedback.
